worksheet_C/README.md

```diff
@@ -0,0 +1,14 @@
+```
+a) This algorithm checks a list for duplicate values
+b) For every list item, the algorithm runs though every list item, so if it has to go through all of them
+it is n^2
+c) The (i-1)th item in the list has already been checked against all of the other items in the list, so it
+doesn't need to be checked again.
```

Comment:
Not quite. The idea is that the ith element is only checked against the elements before it in the list, not the ones after it.